src/jabs/feature_extraction/base_features/angles.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, poses: PoseEstimation, pixel_scale: float):
         self._num_angles = len(AngleIndex)
 
     def per_frame(self, identity: int) -> np.ndarray:
```

Comment:
This type seems to be wrong. I think it's a `dict[str, np.ndarray]`.

Reply:
yup, there are a bunch of these that were wrong that I fixed (these functions used to return ndarrays but then changed to dicts and most of the return type hints were not properly updated)